### ISSUE TITLE
Qt 5.7 VS 2015 32-bit

### ISSUE THREAD

**Issue by FeodorFitsner** [appveyor/ci#2502]

**Comment by pbek** [comment]
+1

**Comment by pbek** [comment]
Thanks a lot, @FeodorFitsner!

Do I need to use a different image or something in <https://github.com/pbek/QOwnNotes/blob/develop/appveyor.yml>, because the Qt 5.7 build still fails (https://ci.appveyor.com/project/pbek/qownnotes/build/job/gjdc8uxw0p7avrpr).

**Comment by FeodorFitsner** [comment]
We've installed VS 2015 32-bit flavor (`C:\Qt\5.7.0\msvc2015` folder).

**Comment by pbek** [comment]
Thank you for your help! I switched the folder, but it's still not building, not even `qmake` can be found (https://ci.appveyor.com/project/pbek/qownnotes/build/windows-b3695/job/4t6icfh7kfi520qf). I guess I'm doing still something wrong, does I?

**Comment by pbek** [comment]
Yes, it is depending on MinGW. And I didn't even know that you can RDP in a build VM (https://www.appveyor.com/docs/how-to/rdp-to-build-worker/). :) I guess I will have to play around with that... Thank you for helping!

**Comment by pbek** [comment]
I found out the folder is actually `C:\Qt\5.7\msvc2015`. But while building I got tons of `undefined reference` errors in <https://ci.appveyor.com/project/pbek/qownnotes/build/windows-b3699/job/e8pm2dwkxmjat3bp>.

**Comment by FBachofner** [comment]
Thanks @FeodorFitsner and @pbek for staying on top of this issue.
I really appreciate your efforts!

**Comment by FBachofner** [comment]
Hi @FeodorFitsner

Is there any chance you can look into the build errors @pbek is experiencing with Appveyor / QT 5.7?

We are very close to giving up hope on getting updated hardware drivers for some legacy equipment and can't (maybe ever) move at least a couple PCs to Windows 10 (at least until the old equipment fails and can't be repaired).

These few machines are networked only internally and QOwnNotes is quite helpful to the users involved . . . but we can not run current QOwnNotes versions based on QT 5.10 on Windows XP    :-(

Thanks in advance for any help you can provide.

**Comment by FeodorFitsner** [comment]
It would be great if you guys could confirm that MinGW build of Qt 5.7 is required to build your projects, so we can re-install Qt. To do that you can [login via RDP](https://www.appveyor.com/docs/how-to/rdp-to-build-worker/) during the build and install Qt 5.7 MinGW using Qt Maintenance Tool, then continue the build.

**Comment by FBachofner** [comment]
Hi @FeodorFitsner 

Thanks for your quick reply!

Earlier in this thread, QOwnNotes developer @pbek stated:
> Yes, it is depending on MinGW

By referencing him I'm sure he will see your instructions.

**Comment by FeodorFitsner** [comment]
Added a new issue: https://github.com/appveyor/ci/issues/2556

**Comment by FBachofner** [comment]
@FeodorFitsner

Thanks for keeping "Qt 5.7 MinGW on VS 2015 image" alive on appveyor!

> By referencing him I'm sure he will see your instructions.

It looks like @pbek finally saw this (he made a comment at #2556 a couple days ago) and (the Win XP version of) his application again builds properly.

I probably should have pointed out this thread to him and he would have had earlier results!
